### DISCUSSION TITLE
I filled up USER_SESSION_STRING wrong

### DISCUSSION THREAD

**Discussion by P4RZIVAL286** [anasty17/mirror-leech-telegram-bot#1225] [Q&A] [answered]
I filled up the user session string wrong from the bot pm settings, now the bot  doesn't deploy, changing the string in config.env doesnt help , where should i change/remove the string for the bot to read and deploy correctly 
![Screenshot (50)](https://user-images.githubusercontent.com/105545318/202645128-0a4fd2b8-97bb-46aa-9308-b2a83bd0e3b7.png)

**Comment by anasty17** [comment, marked as answer]
U can edit database dict from browser! Cluster --> browse collections.
U can reset database if u don't mind.
